### DISCUSSION TITLE
How to configure quarto to use a single copy of index_files when making several revealjs slides?

### DISCUSSION THREAD

**Discussion by UserNameUnavailableIsUnavailable** [quarto-dev/quarto-cli#11728] [Q&A]
### Description

I am deploying my own site to store all my revealjs slides made by quarto. When using `quarto render` option to render `index.md`, the output `index.html` file comes with an `index_files` folder, containing dependencies needed by `index.html`, which are typically 5~6 MB, much bigger than the slideshow itself.

![image](https://github.com/user-attachments/assets/7486f07d-5d95-4fd9-82d6-2c94b83386f8)

I wonder if it is possible to configure quarto so that all output html files will reference to a global single copy of `index_files` to save disk space.

Currently I deal with this issue by regex replacing all `href="index_files/..."` (or `src="index_files/..."`) with the url of the single copy of `index_files` in my server, but this does not resolve the issue that quarto generates redundant `index_files` for each `.md` file in my PC, and I have to write a script to delete them.

**Comment by mcanouil** [comment]
That's automatically done if you use a Quarto project.

**Reply by UserNameUnavailableIsUnavailable** [reply]
Thanks for your reply. I check the docs more carefully and realize that I confused revealjs with a type of project. I should specify the `type` field as `website` 🙃.
